dataland-frontend/src/components/general/TheQuotes.vue

```diff
@@ -0,0 +1,230 @@
+<template>
```

Comment:
This component seems not to be completely finished yet (some console.logs and commented out code in it). 

Furthermore, I have the feeling that there might be a lot of duplicate code for the drag functionality, which seems very similar to the same code in the `TheHowItWorks` component.
I have a feeling that we could centrally define this drag functionality, and then reuse it in both components.

Reply:
Yes, correct. TheQuotes component is still in progress and waiting for content.
In the next version I will create a composable with all the logic of the slider to reuse it in both components.

Reply:
There is a new version now created with the big help of @fsoffel which uses a composable shared with the two sliders